core/src/androidUnitTest/kotlin/com/datadog/kmp/DatadogTest.kt

```diff
@@ -0,0 +1,197 @@
+import com.datadog.kmp.privacy.TrackingConsent
+import com.datadog.kmp.tools.config.ApplicationContextTestConfiguration
+import com.datadog.kmp.tools.config.TestConfiguration
+import com.datadog.kmp.tools.forge.Configurator
+import com.datadog.kmp.tools.unit.annotations.TestConfigurationsProvider
+import com.datadog.kmp.tools.unit.extensions.TestConfigurationExtension
+import fr.xgouchet.elmyr.annotation.AdvancedForgery
+import fr.xgouchet.elmyr.annotation.Forgery
+import fr.xgouchet.elmyr.annotation.MapForgery
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.annotation.StringForgeryType
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.times
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.mockito.quality.Strictness
+
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class),
+    ExtendWith(TestConfigurationExtension::class)
+)
+@ForgeConfiguration(Configurator::class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class DatadogTest {
+
+    private val datadogAndroid = mock<com.datadog.android.Datadog>()
+
+    private lateinit var datadog: Datadog
+
+    @BeforeEach
+    fun setUp() {
+        datadog = spy(Datadog).also { spy: Datadog ->
+            whenever(spy.platformImplementation).doReturn(datadogAndroid)
+        }
+    }
+
+    @Test
+    fun `M call initialize on implementation instance W initialize`(
+        @Forgery configuration: Configuration,
+        @Forgery trackingConsent: TrackingConsent
+    ) {
+        // When
+        datadog.initialize(
+            context = appContext.mockInstance,
+            configuration = configuration,
+            trackingConsent = trackingConsent
+        )
+
+        // Then
+        verify(datadog, times(1))
+            .initialize(
+                context = appContext.mockInstance,
+                configuration = configuration,
+                trackingConsent = trackingConsent
+            )
+
+        assertThat(datadog.isCrashReportingEnabled).isEqualTo(configuration.coreConfig.trackCrashes)
+    }
+
+    @Test
+    fun `M throw IllegalArgumentException W initialize { context is null}`(
```

Comment:
```suggestion
    fun `M throw IllegalArgumentException W initialize { context is null }`(
```

Reply:
the formatting issue is still there

Reply:
than's wired, I've applied `ktlint -F` before push

Reply:
this won't be covered by `ktlint`, because it is still a part of the string literal which is a function name.